### PR TITLE
Add explanation about AMPByExample CSS

### PR DESCRIPTION
**Pull request by kul3r4**
There is some confusion about how AMPByExample manages the CSS for the sample pages. This is an attempt on improving the documentation.
@sebastianbenz @juliantoledo @borisf

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,6 +148,13 @@ You can mark samples as drafts if they are still work-in-progress. This means th
 }--->
 ```
 
+#### How to style examples
+
+You can’t reference external stylesheets when creating AMP pages. All styles must live in the head of the document using the tag `<style amp-custom>`.
```
Comment (sebastianbenz) [inline README.md]: * s/creating AMP pages/creating samples/
* add: "AMP by Example provides a default styling for common elements (p, h1, h2, h3, a, ...) which you should use."

File: README.md [hunk 2]
```diff
@@ -148,6 +148,13 @@ You can mark samples as drafts if they are still work-in-progress. This means th
...
+For AMPByExample, we are using templates, so that we can manage better the CSS, as suggested [here](https://www.ampproject.org/docs/guides/responsive/style_pages.html).
```
Comment (sebastianbenz) [inline README.md]: remove this sentence

File: README.md [hunk 3]
```diff
@@ -148,6 +148,13 @@ You can mark samples as drafts if they are still work-in-progress. This means th
...
+After you write the example and the page is rendered by gulp, the CSS classes will be added to the AMP page from the existing CSS files.
```
Comment (sebastianbenz) [inline README.md]: remove this sentence

File: README.md [hunk 4]
```diff
@@ -148,6 +148,13 @@ You can mark samples as drafts if they are still work-in-progress. This means th
...
+If you copy the example directly from the code section, you will not get the style that you can see in the demo section. We try to limit the CSS that we add to the examples only adding the relevant one for the specific AMP component that we are testing; all the other required CSS will be added after the template is rendered.
```
Comment (sebastianbenz) [inline README.md]: s/after the template is rendered/by the AMP by Example default styling.

Comment (sebastianbenz) [inline README.md]: suggestion: rephrase first sentence to

"Please note: if you copy code from a sample's code section, you will not get the style that you can see in the preview section."

File: README.md (hunk 2 above)
Comment (kul3r4) [inline README.md]: We are using templates, in the sense that the css is added when the template is rendered, like what is happens here:https://github.com/ampproject/amp-by-example/blob/master/templates/example.html

File: README.md (hunk 3 above)
Comment (kul3r4) [inline README.md]: I am trying to say what this file does https://github.com/ampproject/amp-by-example/blob/master/tasks/compile-example.js in the compileExamples function.

File: README.md (hunk 1 above)
Comment (kul3r4) [inline README.md]: ok

File: README.md (hunk 4 above)
Comment (kul3r4) [inline README.md]: ok

**Comment by kul3r4** [comment]
@sebastianbenz what do you think about this pull request? If you don't think it makes sense, I can close it.

File: README.md [hunk 5]
```diff
@@ -148,6 +148,13 @@ You can mark samples as drafts if they are still work-in-progress. This means th
 }--->
 ```
 
+#### How to style examples
+
+You can’t reference external stylesheets when creating samples. AMP by Example provides a default styling for common elements (p, h1, h2, h3, a, ...) which you should use. All styles must live in the head of the document using the tag `<style amp-custom>`.
```
Comment (sebastianbenz) [inline README.md]: can you make "default styling" link to styles.css

File: README.md [hunk 6]
```diff
@@ -148,6 +148,13 @@ You can mark samples as drafts if they are still work-in-progress. This means th
...
+For AMPByExample, we are using templates, so that we can manage better the CSS, as suggested [here](https://www.ampproject.org/docs/guides/responsive/style_pages.html).
```
Comment (sebastianbenz) [inline README.md]: I'd remove this sentence.

File: README.md (hunk 5 above)
Comment (sebastianbenz) [inline README.md]: s/All styles must live in the head of the document using the tag/Sample specific styles must live in the head of the document using the tag

File: README.md [hunk 7]
```diff
@@ -148,6 +148,13 @@ You can mark samples as drafts if they are still work-in-progress. This means th
...
+After you write the example and the page is rendered by gulp, the CSS classes will be added to the AMP page from the existing CSS files.
```
Comment (sebastianbenz) [inline README.md]: Suggestion, rephrase to: If you compile a sample via Gulp and run it, the default styling will be applied.

**Comment by sebastianbenz** [comment]
Sorry  for not responding. I think your additions absolutely make sense. I've added a few suggestions, other LGTM

**Comment by kul3r4** [comment]
@sebastianbenz PTAL

**Comment by sebastianbenz** [comment]
LGTM

**Pull request merged**